Microsoft/WebTemplateStudio
# Add Mock VSCode API for Pages

Task: https://microsoftgarage.visualstudio.com/Intern%20GitHub/_workitems/edit/30767

**Changes**
- Add Mock VS code API data for pages so that pages can be retrieved when the application is run in the browser.

**How to Check**
- Navigate to `src/client`
- Run `yarn start`
- Go to `localhost:3000`
- Navigate through the wizard
- You should be able to see the pages

**Note**: The pages have been defaulted to React pages.
